components/navbar/Navbar.js

```diff
@@ -74,7 +74,7 @@ export default function Navbar() {
                         style: { verticalAlign: "middle" },
                       }}
                     >
-                      <FaGithub />
+                      <FaGithub aria-label=“GitHub”/>
```

Comment:
Change the quotes to the standard double quote. The stylized double quotes that iOS uses when I typed in Discord are causing React to fail.